README.yaml

```
@@ -0,0 +1,73 @@
+---
+#
+# This is the canonical configuration for the `README.md`
+# Run `make readme` to rebuild the `README.md`
+#
+
+# Name of this project
+name: terraform-aws-lambda-elasticsearch-cleanup
+
+# Logo for this project
+#logo: docs/logo.png
+
+# License of this project
+license: "APACHE2"
+
+# Canonical GitHub repo
+github_repo: cloudposse/terraform-aws-lambda-elasticsearch-cleanup
+
+# Badges to display
+badges:
+  - name: "Build Status"
+    image: "https://travis-ci.org/cloudposse/terraform-aws-lambda-elasticsearch-cleanup.svg?branch=master"
+    url: "https://travis-ci.org/cloudposse/terraform-aws-lambda-elasticsearch-cleanup"
+  - name: "Latest Release"
+    image: "https://img.shields.io/github/release/cloudposse/terraform-aws-lambda-elasticsearch-cleanup.svg"
+    url: "https://github.com/cloudposse/terraform-aws-lambda-elasticsearch-cleanup/releases/latest"
+  - name: "Slack Community"
+    image: "https://slack.cloudposse.com/badge.svg"
+    url: "https://slack.cloudposse.com"
+
+related:
+  - name: "terraform-aws-vpc"
+    description: "Terraform Module that defines a VPC with public/private subnets across multiple AZs with Internet Gateways"
+    url: "https://github.com/cloudposse/terraform-aws-vpc"
+  - name: "terraform-aws-dynamic-subnets"
+    description: "Terraform module for dynamic subnets provisioning."
+    url: "https://github.com/cloudposse/terraform-aws-dynamic-subnets"
+  - name: "terraform-aws-elasticsearch"
+    description: "Terraform module for AWS ElasticSearch provisioning."
+    url: "https://github.com/cloudposse/terraform-aws-elasticsearch"
+
+# Short description of this project
+description: |-
+  Terraform module to provision a scheduled Lambda function which will 
+  delete old Elasticsearch indexes using SigV4Auth authentication. The
+  lambda function can optionally send output to an SNS topic if the 
+  topic ARN is given. This module was largely inspired by 
+  [aws-lambda-es-cleanup](https://github.com/cloudreach/aws-lambda-es-cleanup)
+
+# How to use this project
+usage: |-
+  ```hcl
+  module "elasticsearch-cleanup" {
```

```suggestion
  module "elasticsearch_cleanup" {
```

use snake case to name resources